pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -712,6 +723,42 @@
       %^  return-static-data-on-duct  404  'text/html'
       (error-page 404 authenticated url.request ~)
     ==
+ ::  +handle-cache-req: respond with cached value, 404 or 500
+  ++  handle-cache-req
+    |=  [authenticated=? =request:http =path]
+    |^  ^-  (quip move server-state)
+    ?.  authenticated
+      (error-response 403 ~)
+    ?.  =(%'GET' method.request)
+      (error-response 405 "may only GET scries")
+    =/  cache-entries  (fall (~(get of cache.state) path) ~)
+    ?~  cache-entries
+      (error-response 404 "no cache entry for that binding")
+    =*  cache-entry  i.cache-entries
+    ?-    -.cache-entry
+        %payload
+      %-  handle-response
+      :*  %start
+          response-header.simple-payload.cache-entry
+          data.simple-payload.cache-entry
+          complete=%.y
+      ==
+        %scry
+      =/  res=(unit (unit cage))  (rof ~ %cx [our %base ud+case.cache-entry] path.cache-entry)
```

Comment:
this only supports scrying from Clay btw. I assume we want to support any kind of scry, but I'm not sure how to go about that.

Reply:
That's fine for now.